packages/forma-36-react-components/src/components/Asset/AssetIcon/README.mdx

```diff
@@ -0,0 +1,79 @@
+import { AssetIcon } from './AssetIcon';
+
+# AssetIcon
+
+The AssetIcon component is a visual representation of an asset type such as an archive or a video.
```

Comment:
@denkristoffer What do you think if we mention where this component is used, by which components, and that it is a subcomponent? It is a subcomponent, right?